service/src/main/java/org/cbioportal/genome_nexus/service/cached/BaseCachedExternalResourceFetcher.java

```diff
@@ -121,7 +118,7 @@ public T fetchAndCache(String id) throws ResourceMappingException
         }
     }
 
-    public List<T> fetchAndCache(List<String> ids) throws ResourceMappingException
+    public Map<String, T> constructFetchedMap(List<String> ids) throws ResourceMappingException
```

Comment:
We may want to add this method signature to the base interface `CachedExternalResourceFetcher`

Reply:
@onursumer added!

Reply:
@onursumer added!